crates/detect-wasi/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "detect-wasi"
+description = "Detect if WASI can be run"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/detect-wasi"
+version = "1.0.0"
+rust-version = "1.61.0"
+authors = ["Félix Saparelli <felix@passcod.name>"]
+edition = "2021"
+license = "Apache-2.0 OR MIT"
```

Comment:
@passcod I think we should have separate `LICENSE.txt`s for each workspace, instead of having a single `LICENSE.txt` at the root of the repository, which is confusing since not all of them is licensed under GPL.